src/components/PhotoGallery.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component, PropTypes } from 'react'
+import GridTile from 'material-ui/lib/grid-list/grid-tile'
+import FontIcon from 'material-ui/lib/font-icon'
+import Colors from 'material-ui/lib/styles/colors'
+import Checkbox from 'material-ui/lib/checkbox'
+import Sortable from 'react-sortablejs'
+
+// Components
+import Modal from './Modal.jsx'
+
+const styles = {
+  deleteIcon: {
+    cursor: 'pointer'
+  },
+}
+
+const renderContent = (props) => {
+  if(props.editable) {
+    return props.photos.map((photo, index) => {
+      return (
+        <GridTile
+          key={index}
+          data-id={index}
+          titleBackground={'#ccc'}
+          title={
+            <FontIcon
+              className="material-icons"
+              style={styles.deleteIcon}
+              onClick={props.handleDelete.bind(this, photo.id)}
+              color={Colors.red800}
+            >
+              delete
+            </FontIcon>
+          }
+          actionIcon={
+            <Checkbox
+              label="Check"
+              checked={photo.checked}
+              onCheck={props.checkItem.bind(this, index, photo.checked)} />
+          }>
+            <a href={photo.userImageURL} target="_blank">
+              <img src={photo.userImageURL} />
+            </a>
+        </GridTile>
+      )})
+  } else {
+      return props.photos.map((photo,index) => {
+        return (
+          <GridTile
+              data-id={index}
+              key={index}
+              titleBackground={'#ccc'}
+           >
+            <a href={photo.userImageURL} target="_blank">
+            <img src={photo.userImageURL} />
+            </a>
+          </GridTile>
+        )
+      })
+  }
```

Comment:
It looks like the `GridTitle.title` is the only difference between the `props.editable` conditional. If that's the case, could you DRY this up a bit by conditionally setting a different `title` value?